### ISSUE TITLE
Tuxedo Tomte keeps on displaying notification continuously

### ISSUE THREAD

**Issue by petrisch** [tuxedocomputers/tuxedo-control-center#106]
Hi Tuxedo Team

My Aura with Budgie keeps on displaying "Tuxedo Tomte is installing fixes", and "Tuxedo Tomte is finished" repeatedly over again.
I did the normal sudo apt upgrade today.

Not shure if its really doing something...

**Comment by petrisch** [comment]
Turns out probably the apt process got interrupted for some reason.
`sudo dpkg --configure -a`
did the job

**Comment by Emohr-Tuxedo** [comment]
Hello petrisch, sorry for the late answer, I am one of the developers and we want to improve Tomte as much as possible. I am curious about what could have caused your problem. If you mind, could you please send me the file tomte.log located in /var/log/tomte/ ?

There is no personal information there, just stuff like when Tomte starts and what it is doing, like installing some fix and what happened afterwards. If you want you could take a look at it yourself. It is kept quite simple.

Now, I would like to take a look at it to see where it got stuck and to program a way around it. If you want you can send me the file directly to e.mohr@tuxedocomputers.com

Thanks in advance and have a nice time.

**Comment by frankschmitt** [comment]
Same problem here. I've attached the log excerpt from today.

[tail-tomte.log](https://github.com/tuxedocomputers/tuxedo-control-center/files/15137949/tail-tomte.log)

**Comment by frankschmitt** [comment]
I tried uninstalling and re-installing the Nvidia drivers as suggested and ended up with an unusable installation - after the initial Tuxedo logo and the beginning of the boot process, the screen stayed black.

After several tries with the Grub shell (available via the ESC key), I finally managed to get into the Grub menu (with SHIFT), booted in failsafe mode and was able to repair the installation. The good news is: the notification is gone :smile: 

@Emohr-Tuxedo Thanks for your help!
